agent_application/Cargo.toml

```diff
@@ -16,6 +16,7 @@ agent_verification = { path = "../agent_verification" }
 axum.workspace = true
 did_manager.workspace = true
 identity_document = { version = "1.3" }
+identity_verification = { version = "1.3" }
```

Comment:
This in `did-manager` introduced an openssl dependency in UniMe before. Although there doesn't seem to be any issue here, maybe it is smart to pre-emptively set this to `identity_verification = { version = "1.3", default-features = false }` so we don't run in the same issue again once we add UniCore to UniMe.

(perhaps also smart to add it to the workplace.dependencies since `agent_shared` also depends on it)